src/Http/Controllers/Controller.php

```diff
@@ -225,7 +225,11 @@ function (Constraint $constraint) {
             /********** TIMESTAMP TYPE **********/
             case 'timestamp':
                 if ($request->isMethod('PUT')) {
-                    $content = gmdate('Y-m-d H:i:s', strtotime($request->input($row->field)));
+                    if (null === $request->input($row->field)) {
```

Comment:
Please check for empty instead, since we also support Laravel 5.3, which will have this as a empty string instead.
The `empty` function will catch both empty string and `null`.